server/rpc.go

```diff
@@ -210,13 +213,24 @@ func getTranscoder(context context.Context, orch Orchestrator, req *net.Transcod
 	for i, s := range sids {
 		stringStreamIds[s.String()] = job.Profiles[i].Name
 	}
+	mid, err := core.StreamID(job.StreamId).ManifestIDFromStreamID()
+	if err != nil {
+		// XXX what to do?
+		glog.Errorf("Error creating manifest id: %v", err)
+	}
+	smid := string(mid)
 
 	tr := net.TranscoderInfo{
 		Transcoder:  orch.ServiceURI().String(), // currently,  orchestrator == transcoder
 		AuthType:    AuthType_LPE,
 		Credentials: creds,
 		StreamIds:   stringStreamIds,
 	}
+	if drivers.ExternalStorage != nil {
+		storageSession := drivers.ExternalStorage.StartSession(req.JobId, smid, 0)
```

Comment:
We seem to be potentially starting an "external storage" session multiple times, once here and in `ServeSegment`

Reply:
I'll remove session. Purpose of code here is to create policy to send to broadcaster, so B can write into T's S3

Reply:
Can we be explicit about creating the policy / credentials? To keep the generic OS interface simple for now, we can assume "ExternalStorage" to be a concrete S3 driver with a particular `GeneratePolicy` function. Will that work here?